setup.py

```diff
@@ -47,5 +47,4 @@
     zip_safe=zip_safe,
     entry_points=entry_points,
     classifiers=["License :: OSI Approved :: BSD License"],
-    python_requires='>=2.7.11,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*'
```

Comment:
I think this works now on older versions of python... We'll add back the version requirements if we find any issues